lib/board.rb

```diff
@@ -128,26 +130,32 @@ def valid_vertical_placement_length_three?
     end
   end
 
-  def render(reveal=false)
+  def create_header_row
+    dimensions = @cells.keys.last[1].to_i
+
+    numbers = ("1".."#{dimensions}").to_a.join(" ")
+
+    header_row = "  #{numbers}\n"
+    print header_row
+  end
 
-    " 1 2 3 4 \n" +
-    "A #{cells["A1"].render(reveal)} #{cells["A2"].render(reveal)} #{cells["A3"].render(reveal)} #{cells["A4"].render(reveal)} \n" +
-    "B #{cells["B1"].render(reveal)} #{cells["B2"].render(reveal)} #{cells["B3"].render(reveal)} #{cells["B4"].render(reveal)} \n" +
-    "C #{cells["C1"].render(reveal)} #{cells["C2"].render(reveal)} #{cells["C3"].render(reveal)} #{cells["C4"].render(reveal)} \n" +
-    "D #{cells["D1"].render(reveal)} #{cells["D2"].render(reveal)} #{cells["D3"].render(reveal)} #{cells["D4"].render(reveal)} \n"
-
-#FOR REFACTOR SEASON
-    # render_hash_accessible = @cells.reduce(Hash.new {|h,k| h[k] = []}) do |render_hash, cell|
-    #   letter = cell[0].split(//)[0]
-    #   render_hash["#{letter}"] << "#{cell[1].render(reveal)}"
-    #   render_hash
-    # end
-    #
-    # render_hash_accessible.each do |letter, cell_array|
-    #   print "#{letter}"
-    #     cell_array.map { |rendered_cell| print " #{rendered_cell}"}
-    #       puts " "
-    # end
+  def render_alphabetic_hash(reveal=false)
```

Comment:
Again, love this. It is very clear that this method is for the alpha part of the board.